### PR TITLE
Support buildNumber replacing versionCode

### PR DESCRIPTION
**Pull request by rampr**

### CODE REVIEW

**Comment by sksethi25** [comment]
Awesome :+1:

**Comment by zephyr2k** [comment]
:+1:

File: build.js [hunk 1]
```diff
@@ -126,10 +126,11 @@ function updateInfoPlist(app, config, plist) {
   var version = (manifest.ios && manifest.ios.version) ||
       manifest.version ||
       '0.0.0';
-  var versionCode = (manifest.ios && manifest.ios.versionCode) || version;
+  var buildNumber = (manifest.ios && manifest.ios.buildNumber) || version;
```
Comment (bijoshtj) [inline build.js:129]: is this ios only part??

Comment (rampr) [inline build.js:129]: yeah, this project is only ios

**Pull request merged**